app/i18n/locales/paper-wallets/intro/en-US.md

```diff
@@ -10,5 +10,5 @@ However, you should stop using a paper wallet after you restore its funds as it
 
 Yoroi Paper Wallets are protected with a **custom user password** and contain 21 secret words.
 Even if someone gets access to this paper-wallet, they will also need to know your password to gain access to the funds.
-**But if you lose or forget your password, your funds will be lost forever and no one will be able to restore them!**.
```

Comment:
@sarveshwar-singh how was the process to update this?

Reply:
@nicarq I assume it's because of double punctuation. In production it currently says 
> no one will be able to restore them!.

Reply:
@nicarq @SebastienGllmt 
`!.` -> `.`
Yes, double punctuation.
do we want separate PR for this change?

Reply:
@sarveshwar-singh no! it’s okay.